### ISSUE TITLE
[a11y]: Combobox - The scrollable element <ul> with non-interactive content is not tabbable

### ISSUE THREAD

**Issue by oartigue** [carbon-design-system/carbon#16556]
### Package

@carbon/react

### Browser

Chrome, Firefox

### Operating System

MacOS, Other

### Package version

1.58.0

### Automated testing tool and ruleset

IBM Accessibility checker

### Description

The list box of the Combobox doesn't have tabIndex. So the IBM accessibility checker returns this error:
`The scrollable element <ul> with non-interactive content is not tabbable`

### Reproduction/example

See Steps to reproduce

### Steps to reproduce

* Go to https://react.carbondesignsystem.com/?path=/story/components-combobox--playground
* Inspect the `<ul role="listbox" class="cds--list-box">` element, you can see there is no tabIndex

Notes: 
* the button of combobox have got an tabIndex.
* I found this issue for dropdown (https://github.com/carbon-design-system/carbon/issues/13110) which had been met the same problem. I didn't find any same issue for combobox.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by LUCROFT** [comment]
Hi ! Sorry to hassle but for our AVT deviation, it would be great to have a timeframe for a fix. Any ideas ? Thanks.

**Comment by guidari** [comment]
Hey, I just added to our Backlog

**Comment by LUCROFT** [comment]
Hello @guidari ! Any news on this work item ? Thanks.

**Comment by guidari** [comment]
Hey @mbgower 
Just following up on our conversation from Monday (Feb 3).
Did you check with the accessibility team about this error in ComboBox? It doesn’t seem like an actual error, since it is an expected behaivour from the component to no accept Tab navigation.

**Comment by guidari** [comment]
This issue was passed to the checker team. We concluded that this is not an error but the expected behavior of the component, which only supports arrow navigation instead of Tab navigation.
@mbgower sent it to the checker team.
